src/visualizers/Grid.ts

```diff
@@ -633,6 +633,10 @@ earlier ones that use the _same_ style.)
     }
 
     setup(): void {
+        super.setup()
+        if (!this.sketch) {
+            throw 'Attempt to show Grid before injecting into element'
```

Comment:
Some of the consistent changes happening here, I wonder if there should be any accompanying documentation.  I know we don't have properly documented info on how to build visualizers yet, but this construction here for example may need some explanation somewhere?  Not sure if that's in the scope of this PR -- probably more appropriate to create a nicely commented/documented example visualizer template.  So I'll leave this comment here for now and we can maybe open an issue.

Reply:
I will try to say something brief in the doc review/commit I am about to do, and then hopefully we can get more on this in a ExampleP5Visualizer.ts that Aaron may have time to do tomorrow. (Or even better if one of you figure out a cleverer way to do this so that setup can be written more smoothly in the Visualizers derived from P5Visualizer...)

Reply:
Just something to go to @Vectornaut for the example visualizer, probably.

Reply:
Oops, we are overlapping in my ability to respond (and consider the options here)... see my response to the next code comment below for some other options here that will likely make the derived Visualizers code look/feel noticeably smoother, and let me know what you think.